src/devices/openxrheadset/OpenXrInterface.cpp

```diff
@@ -1704,36 +1832,59 @@ void OpenXrInterface::getAllPoses(std::vector<NamedPoseVelocity> &additionalPose
         }
     }
 
+    numberOfPoses += 52;  // adding also fingers
```

Comment:
Please avoid hardcoded numbers. In principle here you know if the hand tracking is supported, and the number of fingers too